getting socket from backbone objects

dodo opened the pull request:
these commits allows to set different sockets for models or collections simply by doing:

``` javascript
IOModel = Backbone.Model.extends({
    defaults:{touched:0},
    socket:window.socket, // <------------ this is new
    initialize: function () {
        this.ioBind('finger', this.touch, this);
    },
    touch: function () {
        this.set({touched:this.get('touched')+1});
    },
});
```

dodo committed:
get socket from backbone objects as well so they all can have their own socket

dodo committed:
update example

logicalparadox commented:
Good idea! 

However, instead of modifying the root files directly, can you make the changes in the lib/model.js, [...] then run the build tool?

dodo commented:
ups  mD

dodo committed:
*fail*
Revert "get socket from backbone objects as well so they all can have their own socket"

This reverts commit f4c194d6ccf8eb7eb2f6ce9a8d7c6187e2abed45.

dodo committed:
get socket from backbone objects as well so they all can have their own socket ... now for real

dodo commented:
fixed.

logicalparadox commented:
Thanks so much. Let me know if you have any more ideas, etc...
